app.py

```diff
@@ -72,11 +72,18 @@
     logging.warning('Cannot determine release; enable Dyno metadata by running "heroku labs:enable runtime-dyno-metadata -a <APP_NAME>"')
 
 
-@app.route('/parse/', methods=['GET'])
+@app.route('/parse', methods=['POST'])
 def parse():
-    # Retrieve the name from url parameter
-    code = request.args.get("code", None)
-    level = int(request.args.get("level", None))
+    body = request.json
+    if not body:
+        return "body must be an object", 400
```

Comment:
This looks weird. Are you sure this is the way to return a 400 response?

Reply:
Good question. I just checked and it does seem to return the string as the body and the integer as the response code. I found this line in this official doc (http://www.flaskapi.org/api-guide/status-codes/): `return content, status.HTTP_404_NOT_FOUND`